src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -83,18 +88,18 @@ func (a *StorageMinerActorCode_I) _isChallenged(rt Runtime) bool {
 }
 
 // called by CronActor to notify StorageMiner of PoSt Challenge
-func (a *StorageMinerActorCode_I) NotifyOfPoStChallenge(rt Runtime) {
-	rt.ValidateCallerIs(vmr.CronActorAddr)
+func (a *StorageMinerActorCode_I) NotifyOfPoStChallenge(rt Runtime) InvocOutput {
+	rt.ValidateCallerIs(addr.CronActorAddr)
```

Comment:
this shouldn't be coming from `addr`. runtime is a much better place to do it -- we can use a file like `exitcodes`